apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -122,6 +122,14 @@ def get_string_precision(number_string: str) -> float:
 
 
 # TODO: add precision calculation function for other grading methods
```

Comment:
I think this TODO can be removed now. Also, it would be great to have independent test cases for these functions. To do that, these need to be put in a separate file, see the `pl-code` element for an example.

Reply:
This is added in the latest commits [12cacd8](https://github.com/PrairieLearn/PrairieLearn/pull/8435/commits/12cacd8af1c4ab5ff2514ca2a4ad745025df9eb4).